test/test_operators.py

```diff
@@ -286,6 +286,23 @@ def reshape(input):
         ref_out, tvm_out = self.runBoth(reshape, input)
         assert torch.allclose(ref_out, tvm_out, rtol=0.01, atol=0.01)
 
+    @TVMTest.given(
+        shape=TVMTest.rand_shape(rank=2, min_dim=4),
```

Comment:
nit: `axis=TVMTest.rand_int(0, 2),`

Reply:
I agree much better to have randomized inputs 👍